api/routes.py

```diff
@@ -185,32 +196,98 @@ def get_sections_for_round_assessment(fund_id, round_id):
     abort(404)
 
 
+def create_event():
+    args = request.get_json()
```

Comment:
How we mange these events ? do we have a dashboard, and cant we manage this using event bridge ?

Reply:
No there's no dashboard, it's used internally (at the moment it's used for sending timed emails like deadline reminders and incomplete applications : https://github.com/communitiesuk/funding-service-design-fund-store/blob/main/db/models/event.py#L16 )